Applications/MaterialManager/Samples/Create/Edgebands/Readme.md

```diff
@@ -0,0 +1,26 @@
+<h1 id="createEdgebandType"> Create edgeband type</h1>
+
+With the Homag Connect materialManager edgebands client, new edgeband types can be created.
+
+<strong>Example:</strong>
+
+```csharp
+// Create new instance of the materialManager client:
+var client = new MaterialManagerClientMaterialEdgebands(subscriptionId, authorizationKey);
+
+// Define the new edgeband type request:
+var edgebandTypeRequest = new MaterialManagerRequestEdgebandType
+{
+    Code = "EB_White_1mm",
+    Description = "White Edgeband 1mm",
```

Comment:
There is no property called "Description"

Reply:
Checked all the properties and required properties.